app/templates/components/dashboard/onboarding/checklist-progress.hbs

```diff
@@ -24,7 +24,7 @@
       </span>
       <span class="step-description">
         {{#if hasRatings}}
-          Rate 5 anime or manga
+          {{t "onboarding.checklist.rate"}}
         {{else}}
           Rate {{ratingsLeft}} anime or manga
```

Comment:
Missing this one and the other else blocks in this file.

Reply:
How can I add that code in the translation file?

Reply:
```js
{{t "example" count=ratingsLeft}}
```

```yaml
example: "Rate {count} anime or manga"
```